pyproject.toml

```diff
@@ -27,8 +27,8 @@ packages = [{ include = "banana", from = "src" }]
 [tool.poetry.dependencies]
 python = "^3.8.0,<3.13"
 ipython = "^8.1.0"
-SQLAlchemy = "^1.4.29,<=2.0"
-numpy = "^1.21.0"
+SQLAlchemy = "^1.4.29"
```

Comment:
That I wouldn't do: https://github.com/N3PDF/banana/issues/22

Reply:
Why?

The `^...` is already meaning *compatible*, i.e. `>=1.4.29,<2.0`, so it is just redundant.